### PR TITLE
screens button + logo panel margin fix

### PR DESCRIPTION
**Pull request by ramtob**

### CODE REVIEW

File: src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less [hunk 1]
```diff
@@ -32,6 +32,10 @@
 
 			i.icon {
 				margin: 0; // override default margin from container
+
+				&:before {
+					content: "\e902"; // override button icon: icon-screen-choice
```
Comment (tltk90) [inline src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less:37]: why not use `icon-screen-choice` class instead change the content

**Review by tltk90 — CHANGES_REQUESTED** [review]

File: src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less (hunk 1 above)
Comment (ramtob) [reply src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less:37]: @tltk90 
Use the class where?
The icon element is inside `ansyn-combo-box`. Not in my component.

Comment (tltk90) [reply src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less:37]: but you pass them to `ansyn-combo-box` through `ansyn-combo-box-option`

Comment (ramtob) [reply src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less:37]: @tltk90 

But I don't want to change the icons of the options. I only change the icon of the top button (.display-button)

Comment (tltk90) [reply src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less:37]: I think that the options should not be icons just text.

Comment (ramtob) [reply src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less:37]: @tltk90  I don't understand. For screens, the options are icons, no?

Comment (tltk90) [reply src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less:37]: I understand from Yuval that the icon only be in the button on the status-bar,
the option on the select menu will be a string

Comment (ramtob) [reply src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less:37]: What kind of string?

Comment (tltk90) [reply src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less:37]: good question.
I ask Yuval

Comment (tltk90) [reply src/app/@ansyn/ansyn/modules/status-bar/components/display-panel/display-panel.component.less:37]: Ok. I don't understand it right.

**Review by tltk90 — APPROVED** [review]

**Pull request merged**